integration/spark/src/test/scala/org/apache/spark/sql/carbondata/datasource/SparkCarbonDataSourceBinaryTest.scala

```diff
@@ -657,6 +657,18 @@ class SparkCarbonDataSourceBinaryTest extends QueryTest with BeforeAndAfterAll {
     || exception.getMessage.contains("DELETE TABLE is not supported temporarily."))
   }
 
+  test("test load with options on parquet table") {
+    sql("drop table if exists parquet_table")
+    sql("create table parquet_table(empno int, empname string, projdate Date) using parquet")
+    val ex = intercept[AnalysisException] {
+      sql(
+        s"""LOAD DATA local inpath '$resourcesPath/data_big.csv' INTO TABLE parquet_table
```

Comment:
can you add a check for load without options also

Reply:
Done